### PR TITLE
First Pull Request 

### PR DESCRIPTION
**Pull request by RideOrDye**
created a new branch for a first pull create.

### CODE REVIEW

File: slider [hunk 1]
```diff
@@ -63,20 +32,56 @@
...
+              <li>50 -</li>
+              <li>40 -</li>
+              <li>30 -</li>
+              <li>20 -</li>
+              <li>10 -</li>
+              <li> 0 - </li>
+          </ul>
+          </div>
+              <div class="range">
+              <div id="slider">
```
Comment (nbeliczay) [inline slider]: please fix indentation

File: slider [hunk 2]
```diff
@@ -63,20 +32,56 @@
...
+               </div>
+            </div>
+        </div>
+        <script>
+      const slider = document.querySelector(`#slider`);
```
Comment (nbeliczay) [inline slider:67]: please fix indentation in the functions included in the `<script>` tag

**Comment by nbeliczay** [comment]
Thank you for adding the screenshot!

**Review by nbeliczay — APPROVED** [review]

**Comment by RideOrDye** [comment]
These changes got merged in pull request #2 (closing this PR)

**Pull request closed without merging**